getting-started/multiband-analysis.rst

```diff
@@ -132,13 +138,14 @@ The Pipelines determine if a detection in a patch is primary, or not, by whether
 An inner region is a part of a sky map exclusively claimed by one patch.
 
 The flag that indicates whether a source lies in the patch's inner region isn't in the ``deepCoadd_forced_src`` table though.
-Instead, you need to look at the ``deepCoadd_ref`` table made by :command:`mergeCoaddMeasurements.py` in the :ref:`previous tutorial <getting-started-tutorial-merge-coadds>`.
+Instead, you need to look at the ``deepCoadd_ref`` table made as part of the ``coadd_measurement`` pipeline in the :ref:`previous tutorial <getting-started-tutorial-merge-coadds>`.
 
-Begin by using the Butler to get the ``deepCoadd_ref`` dataset for  patch you're analyzing:
+Begin by using the Butler to get the ``deepCoadd_ref`` dataset for  patch you're analyzing.
+You will need to use a where clause because, in this case, we are not asking for a single entity for each entry in the data ID:
 
 .. code-block:: python
 
-   refTable = butler.get('deepCoadd_ref', {'filter': 'HSC-R^HSC-I', 'tract': 0, 'patch': '1,1'})
+   refTable = butler.get('deepCoadd_ref', tract=9813, patch=41, where="band IN ('r', 'i')")
```

Comment:
There is no `where` parameter to `butler.get`. This method only returns one dataset and can't return multiple datasets. If you want multiple you would have to use `butler.registry.queryDatasets` and then use butler.getDirect on the results (maybe storing them in a dict indexed by dataId.  I think the big problem you have here is that the system does not complain about the `where=` line because it thinks it's a dataId and we don't validate it -- there is a ticket to fix this.